locales/en/settings.ftl

```diff
@@ -8,9 +8,16 @@ settings-page-title = { -product-short-name } Settings
 
 ## Breach alert preferences
 
+# Obsolete
 settings-alert-preferences-title = Breach alert preferences
```

Comment:
You can just remove the string altogether; since the string and the references to the string are in the same codebase, and you removed the references from the code in this same PR, there's no risk of suddenly showing `???` in the UI.

```suggestion
```